blocks/folder-blocks/infinite-canvas/FilePicker.tsx

```diff
@@ -1,34 +1,31 @@
 import { tw } from "twind";
 import { ActionList, ActionMenu } from "@primer/react";
-
-import { useState } from "react";
-import { Files, File } from "./index";
+import { TreeNode } from "@githubnext/blocks";
 
 export const FilePicker = ({
   files,
   onFileSelected,
 }: {
-  files: Files;
-  onFileSelected: (file: File) => void;
+  files: TreeNode[];
+  onFileSelected: (path: string) => void;
 }) => {
-  // needed to close the dropdown, which is an uncontrolled detail element
-  const [iteration, setIteration] = useState(0);
```

Comment:
This wasn't necessary on my end, though I might be missing some context or was testing it incorrectly. The dropdown closes fine upon item click; this behavior seems to be the default, looking at the Primer docs.